planet/js/GlobalPlanet.js

```diff
@@ -547,6 +547,13 @@ class GlobalPlanet {
             document.getElementById("global-search").addEventListener("input",  (evt) => {
                 this.searchString = document.getElementById("global-search").value;
                 debouncedfunction();
+                console.log("Entereed global search");
+            });
+
+            document.getElementById("global-search-2").addEventListener("input",  (evt) => {
+                this.searchString = document.getElementById("global-search-2").value;
+                debouncedfunction();
+                console.log("Entereed global search-2");
```

Comment:
same typo... but maybe just remove these console messages. (They won't pass the eslint test in any case.)